troposphere/__init__.py

```diff
@@ -39,6 +39,9 @@
 
 valid_names = re.compile(r'^[a-zA-Z0-9]+$')
 
+if sys.version_info[0] == "3":
```

Comment:
There are a few instances of this added to files. The use of 2to3 should not require these changes. Could you elaborate on why this was required?

Reply:
Not happy this is in the commit.  I had looked at converting the code base to be 2 and 3 compatible without the need for 2to3.  In my mind this would be a first refactor on the road to Python 3 native and I was going to comment in the appropriate issues at a later time.  These were some minor experiments and I'll remove and re-commit later today/tonight when I have time.  My apologies.